t/plugin/elasticsearch-logger.t

```diff
@@ -0,0 +1,182 @@
+#
```

Comment:
Let's add `elasticsearch-logger` to https://github.com/apache/apisix/blob/9b0cc7a3b0ed837e330603304c4a7267ff392faf/conf/config-default.yaml#L324 so CI can be run

Reply:
It's added